src/tests/WebApi.MsBuild.SystemTest.ContractFirst/WebApi.MsBuild.SystemTest.ContractFirst.csproj

```diff
@@ -19,6 +19,10 @@
 
     <ItemGroup>
         <PackageReference Include="Swashbuckle.AspNetCore" Version="6.5.0" />
+        <PackageReference Include="Workleap.OpenApi.MSBuild" Version="0.4.0">
```

Comment:
Seems an artifact of running the system test locally, we want the script to add this reference not having it hardcoded to a specific version

Reply:
Fixed